internal/lib/control_config.go

```diff
@@ -205,34 +204,12 @@ func (c *akoControlConfig) CRDClientset() akocrd.Interface {
 	return c.crdClientset
 }
 
+// CRDs are by default installed on all AKO deployments. So always enable CRD parameters.
+// TODO: Optimise
 func (c *akoControlConfig) SetCRDEnabledParams(cs akocrd.Interface) {
-	timeout := int64(120)
-	_, aviInfraError := cs.AkoV1alpha1().AviInfraSettings().List(context.TODO(), metav1.ListOptions{TimeoutSeconds: &timeout})
-	if aviInfraError != nil {
-		utils.AviLog.Infof("ako.vmware.com/v1alpha1/AviInfraSetting not found/enabled on cluster: %v", aviInfraError)
-		c.aviInfraSettingEnabled = false
-	} else {
-		utils.AviLog.Infof("ako.vmware.com/v1alpha1/AviInfraSetting enabled on cluster")
-		c.aviInfraSettingEnabled = true
-	}
-
-	_, hostRulesError := cs.AkoV1alpha1().HostRules(metav1.NamespaceAll).List(context.TODO(), metav1.ListOptions{TimeoutSeconds: &timeout})
-	if hostRulesError != nil {
-		utils.AviLog.Infof("ako.vmware.com/v1alpha1/HostRule not found/enabled on cluster: %v", hostRulesError)
-		c.hostRuleEnabled = false
-	} else {
-		utils.AviLog.Infof("ako.vmware.com/v1alpha1/HostRule enabled on cluster")
-		c.hostRuleEnabled = true
-	}
-
-	_, httpRulesError := cs.AkoV1alpha1().HTTPRules(metav1.NamespaceAll).List(context.TODO(), metav1.ListOptions{TimeoutSeconds: &timeout})
-	if httpRulesError != nil {
-		utils.AviLog.Infof("ako.vmware.com/v1alpha1/HTTPRule not found/enabled on cluster: %v", httpRulesError)
-		c.httpRuleEnabled = false
-	} else {
-		utils.AviLog.Infof("ako.vmware.com/v1alpha1/HTTPRule enabled on cluster")
-		c.httpRuleEnabled = true
-	}
+	c.aviInfraSettingEnabled = true
```

Comment:
now we are making the flag true, is the function to get this flag really required?